pkg/startup/startup.go

```diff
@@ -627,7 +627,7 @@ func validateIP(ipn string) {
 	for _, i := range ifaces {
 		for _, c := range i.Cidrs {
 			if ipAddr.Equal(c.IP) {
-				log.Infof("IPv%d address %s discovered on interface %s", ipAddr.Version(), ipAddr.String(), i.Name)
+				log.Debugf("IPv%d address %s discovered on interface %s", ipAddr.Version(), ipAddr.String(), i.Name)
```

Comment:
A bit of a bummer that we won't see this message on clusters with default log levels set, especially considering ip autodetection is often a source of customer issues.

Reply:
I don't think this log is very important. It's the "mainline, working" branch, and is surrounded by higher level logs for the "unexpected, maybe not working" scenarios.